client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -543,6 +545,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 			categorization={ categorization }
 			isPremiumThemeAvailable={ isPremiumThemeAvailable }
 			purchasedThemes={ purchasedThemes }
+			hasWoopFeature={ hasWoopFeature }
```

Comment:
This feels overly specific to me. Might it not be better to simply pass the list of active plan features to the design picker? That way the design picker has more data available to support any decision it may need to make.
```suggestion
			currentPlanFeatures={ site?.plan?.features.active ?? [] }
```
Also, I am thinking it may be better to simply rely on the `siteHasFeature()` selector from within the Unified Design Picker. (It looks like the selector is available from the `Site` data store in the `@automattic/data-stores` package.)

Reply:
>This feels overly specific to me.

Agreed!

There are a few other props that only depend on `site`. 
I could pass `site` as a prop and then check those things only when needed.
`isPremiumThemeAvailable`, `verticalId` and `purchasedThemes` props could also be removed in this case (from several components in the hierarchy).

It sounds like a good idea to me, but I'm sure there is a reason for the props to be cherry-picked currently. What do you think?

Reply:
Let's keep them distinct for now to follow the pattern, but I think it would be better to pass in the full list of features rather than just this derived property.

Reply:
Or, more specifically, we can just use code like `currentPlanFeatures={ site?.plan?.features.active ?? [] }` as I suggested above.